lib/Sema/TypeCheckDeclOverride.cpp

```diff
@@ -995,11 +1008,13 @@ static void checkOverrideAccessControl(ValueDecl *baseDecl, ValueDecl *decl,
       !baseHasOpenAccess &&
       baseDecl->getModuleContext() != decl->getModuleContext() &&
       !isa<ConstructorDecl>(decl)) {
-    // NSObject.hashValue was made non-overridable in Swift 5; one should
-    // override NSObject.hash instead.
+    // NSObject.hashValue and NSObject.hash(into:) was made non-overridable in
+    // Swift 5; one should override NSObject.hash instead.
```

Comment:
Nit: `NSObject.hash(into:)` was never `open` in any stdlib release -- we added it as a non-overridable function in Swift 4.2.